assign_rights/templates/head.html

```diff
@@ -6,8 +6,10 @@
 <meta content="width=device-width, initial-scale=1, user-scalable=yes" name="viewport">
 <!-- Bootstrap 4.5.0 -->
 <link rel="stylesheet" href="{% static "bootstrap/css/bootstrap.min.css" %}">
+<!-- Custom styles-->
+<link rel="stylesheet" href="{% static "bootstrap/css/custom.css" %}">
```

Comment:
I think I would prefer if this custom stylesheet were not nested within this bootstrap directory. That would make it easier to just replace the Bootstrap files if we want to upgrade.